rest_framework/relations.py

```diff
@@ -80,6 +80,13 @@ def run_validation(self, data=empty):
             data = None
         return super(RelatedField, self).run_validation(data)
 
+    def field_to_native(self, obj, field_name):
+        # Without PK there is no way to determine any relations to the object
+        if obj.pk is None:
```

Comment:
Please check https://github.com/tomchristie/django-rest-framework/pull/2641 to make sure we don't make another mistake here.